src/composables/request.js

```diff
@@ -111,7 +114,9 @@ const _request = (container, awaitingResponse) => (config) => {
       awaitingResponse.value = false;
 
       logAxiosError(logger, error);
```

Comment:
Would you mind including `logAxiosError()` in the `if`? That's what we do in `requestData`.